lib/workers/bin/run_single_worker.rb

```diff
@@ -107,20 +107,14 @@ def all_role_names
   end
 
   update_options = create_options.dup
```

Comment:
NOTE I'm not sure why we're dup'ing `create_options` here anymore, we used to add in pid / system_uid to `update_options` but not after https://github.com/ManageIQ/manageiq/pull/21068 as far as I can tell

Reply:
Yeah that's good point - there's no need to dup it other than readability (i.e. we care about updates here vs creates)

Reply:
Yeah, 😕  ... I'm not sure why that remained.  Good eye.